### ISSUE TITLE
A persistent info message shows up every time I start a vscode insiders

### ISSUE THREAD

**Issue by yoshiotobe** [hashicorp/vscode-terraform#566]
- vscode-terraform : 2.7.0
- vscode insiders : 1.54.0
- terraform :  0.12.27
- windows : 20H2

Like in the photo below, the same message "Installed terraform-ls 0.13.0" shows up every time I start a vscode insiders.

![terraform2](https://user-images.githubusercontent.com/38850403/108803366-db283680-75dd-11eb-9583-65d2ce442dd6.jpg)

It does not happen with vscode. So, I believe this is a problem specific to vscode insiders.
Is there a way to suppress the message? Or this is a bug?

**Comment by szymonos** [comment]
It happens to me every time I start VSCode (non-insiders) too. Worse, if I open the second instance of VSCode, the installation freezes at the end and I cannot use e.g. formatting.

**Comment by szymonos** [comment]
I've managed to "fix" this by manually downloading language server and setting in VSCode pathToBinary in terraform.languageServer section.

**Comment by aeschright** [comment]
Do you have more than one VSCode instance running when this happens (both insiders and the current release)?

**Comment by yoshiotobe** [comment]
Well, I found out that VSCode has the same problem as well as Insiders. The problem happens in both cases (one and two instances). 
I checked the directory where the language server is installed. When I open VScode or Insiders, the extention replaces the binary even though it's there. The problem is that the extension does not recognize the binary is there. So, it installs it over and over.

![terraform](https://user-images.githubusercontent.com/38850403/109251827-82e87300-782f-11eb-932a-9696345f976d.jpg)
